src/assets/Comments.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"

```

Comment:
When using strings, you should use either single or double quotes all the time. So all imports should use single quotes.

Reply:
ok